src/helpers/order-process/order-process-helper.ts

```diff
@@ -73,8 +84,48 @@ export const removeOrderProcesses = (
     )
   );
 
-export const updateOrderProcess = (
+export const updateOrderProcess = async (
   id: string,
-  data: Partial<OrderProcess>
-): Promise<OrderProcess> =>
-  getOrderProcessApi().updateOrderProcess(id, data) as Promise<OrderProcess>;
+  {
+    before_portfolio_item_id,
+    after_portfolio_item_id,
+    ...data
+  }: Partial<OrderProcess>
+): Promise<OrderProcess> => {
+  const op = await getOrderProcessApi().updateOrderProcess(id, {
+    name: data.name,
+    description: data.description,
+    before_portfolio_item_id: before_portfolio_item_id || '',
+    after_portfolio_item_id: after_portfolio_item_id || ''
+  });
+
+  return op as OrderProcess;
+};
+
+export const addOrderProcess = async ({
+  before_portfolio_item_id,
+  after_portfolio_item_id,
+  ...data
+}: Partial<OrderProcess>) => {
```

Comment:
This return type should look like this:
```TS
tPromise<[OrderProcess, OrderProcess | undefined]
```

There might be only one promise, so the other OrderProcess› might be undefined.